pkgs/by-name/an/ananicy-cpp/package.nix

```diff
@@ -33,7 +34,8 @@ clangStdenv.mkDerivation rec {
       url = "https://gitlab.com/ananicy-cpp/ananicy-cpp/-/commit/6ea2dccceec39b6c4913f617dad81d859aa20f24.patch";
       hash = "sha256-C+7x/VpVwewXEPwibi7GxGfjuhDkhcjTyGbZHlYL2Bs=";
     })
-  ];
+  ]
+  ++ lib.optional wrapperSupport ./match-wrappers.patch;
```

Comment:
Let's just always add it

Reply:
Agreed, is this on upstream patch, by the way? If not, I think it should go to upstream directly.

Reply:
No, I wrote it.

Reply:
> Agreed, is this an upstream patch, by the way? If not, I think it should go to upstream directly.

It's nix specific so idk if upstream would accept it, though we can try.

Reply:
Not all process names end like that, as you can see. 

https://github.com/CachyOS/ananicy-rules/blob/e3638ebb2320202ede9facfc9657cc8af2098a2d/00-default/Services/gvfs.rules#L10

I had to manually add these rules to match `.x-` `.x-w`, `.x-wra` and so on.

All wrapped app's processes have `.` in the beginning. 
I think the rules should run on any process `x` and `.x+.`